src/status_im2/navigation/screens.cljs

```diff
@@ -248,8 +248,8 @@
      :options   {:insets {:top? true}}
      :component wallet-accounts/view}
 
-    {:name      :address-to-watch-edit
```

Comment:
I found "edit" to be a bit misleading. We have "edit" account and so I did not want to confuse with that. here we are creating (or adding) the watch only account.

Reply:
I am working on an issue https://github.com/status-im/status-mobile/issues/17806 related to address watch, and I was also doing changes to the naming :) I push my PR to show the changes I made: https://github.com/status-im/status-mobile/pull/17859/files

I renamed `select-address-to-watch` to `add-address-to-watch` (to match the screen title) and moved `add-watch-only-account` under `add-address-to-watch` namespace and renamed it to `edit-address`. I think that makes it clearer that the editing is for the watch address

Reply:
Sure I'll copy your changes, thanks for the heads up! 🙂

Reply:
Renamed it to `confirm-address`. And PR is ready for review https://github.com/status-im/status-mobile/pull/17859 👍